README.md

```diff
@@ -26,6 +27,7 @@
 | `<gtag>` | `latest-<gtag>`   | Latest available release |
 | `<gtag>` | `2.4.3-<gtag>`    | Cruise control v2.4.3    |
 | `<gtag>` | `2.4.2-<gtag>`    | Cruise control v2.4.2    |
+| `<gtag>` | `2.0.106-<gtag>`  | Cruise control v2.0.106  |
```

Comment:
Would be awesome to get a compatibility matrix between this and the Kafka versions. Or at least a link to an existing one. I would imagine that CC has one already, no?